test/infrastructure/vcsim/controllers/vcsim_controller.go

```diff
@@ -286,7 +240,62 @@ func (r *VCenterSimulatorReconciler) reconcileNormal(ctx context.Context, vCente
 	return nil
 }
 
-func addPreRequisitesForVMIPreconciler(ctx context.Context, c client.Client, config vmoperator.Dependencies) error {
+// dependenciesForVCenterSimulator return a dependency config for a vCenterSimulator.
+// Note: This config uses cluster DC0/C0, datastore LocalDS_0 in vcsim; it also sets up content library
+// and the default namespace (the namespace where workload cluster are going to be deployed) with just
+// what is required to reconcile VirtualMachines with the vm-operator.
+func dependenciesForVCenterSimulator(vCenterSimulator *vcsimv1.VCenterSimulator) *vmoperator.Dependencies {
+	datacenter := 0
+	cluster := 0
+	datastore := 0
+
+	config := &vmoperator.Dependencies{
+		// This is where tilt deploys the vm-operator
+		Namespace: vmoperator.DefaultNamespace,
+
+		VCenterCluster: vmoperator.VCenterClusterConfig{
+			ServerURL:     vCenterSimulator.Status.Host,
+			Username:      vCenterSimulator.Status.Username,
+			Password:      vCenterSimulator.Status.Password,
+			Thumbprint:    vCenterSimulator.Status.Thumbprint,
+			Datacenter:    vcsimDatacenterName(datacenter),
+			Cluster:       vcsimClusterPath(datacenter, cluster),
+			Folder:        vcsimVMFolderName(datacenter),
+			ResourcePool:  vcsimResourcePoolPath(datacenter, cluster),
+			StoragePolicy: vcsimDefaultStoragePolicyName,
+
+			// Those are settings for a fake content library we are going to create given that it doesn't exists in vcsim by default.
+			// It contains a single dummy image.
+			ContentLibrary: vmoperator.ContentLibraryConfig{
+				Name:      "vcsim",
+				Datastore: vcsimDatastorePath(datacenter, datastore),
+				Item: vmoperator.ContentLibraryItemConfig{
+					Name: "vcsim-default-image",
+					Files: []vmoperator.ContentLibraryItemFilesConfig{ // TODO: check if we really need both
```

Comment:
TODO still there by accident or on purpose? :-)

Reply:
it was a left-over, cleaned up